frontend/src/components/course/Links.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="links">
+        <p>
+          <a target=_blank :href="handbookURL">Handbook <i class="material-icons">launch</i></a>
```

Comment:
this is a nice touch

Reply:
Can't decide whether these links should be column or row.